fabric-server-language-v1/src/main/java/net/fabricmc/fabric/impl/server/language/ServerTranslationEntry.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.server.language;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import net.fabricmc.loader.api.metadata.ModDependency;
+import net.fabricmc.loader.api.metadata.ModMetadata;
+
+/**
+ * A utility class for holding translation mappings prior collision resolution.
+ */
+public class ServerTranslationEntry {
```

Comment:
Consider declaring this as a `record`

Reply:
Records' field getter system is rubbish.  
Considering compat MC 1.16 & Java 8.

Reply:
For not using records, sure, that is minor.

> Add `@Environment(EnvType.SERVER)`

Notice that now everything in the dedicated server is part of the client since mojang started including data generator in 1.13, so that annotation shouldn't be added.